webpage/webstreaming.py

```diff
@@ -0,0 +1,143 @@
+from webpage.utils import *
+#from tracker_lib import *
+import threading
+import argparse
+from flask import Flask, url_for, jsonify
+from flask import render_template
+from flask_bootstrap import Bootstrap
+from flask import request, redirect
+#from flask_appconfig import AppConfig
+from webpage.config import *
+from webpage.forms import ConfigForm
+from time import sleep
+from detector3 import Detector
+#from time import time
+select_mode = "automatic"
+
+#t0=time()
+def create_app(det, configfile=None):
+    app = Flask(
+                __name__,
+                template_folder="templates",
+                static_folder="static"
+                )
+
+    Bootstrap(app)
+
+    app.config['SECRET_KEY'] = 'devkey'
+    app.config['RECAPTCHA_PUBLIC_KEY'] = \
+        '6Lfol9cSAAAAADAkodaYl9wvQCwBMr3qGR_PPHcw'
```

Comment:
showing the public key?

Reply:
This is exactly the same as the previous project. I think there's no need to hide it right now.